scripts/report-bundle-size/report-as-a-pr-comment.js

```diff
@@ -159,6 +160,22 @@ function formatSize(bundleSize) {
   return `${(bundleSize / 1024).toFixed(2)} kB`
 }
 
+function getEmojiWithThreshold(percentageChange) {
+  const absChange = Math.abs(percentageChange)
+  const emojiMap = [
+    { threshold: 0.15, positive: '🟢', negative: '🎉' },
+    { threshold: 0.3, positive: '🟠', negative: '🥳' },
+    { threshold: 0.5, positive: '😡', negative: '🤩' },
+    { threshold: Infinity, positive: '⛔', negative: '🏆' },
+  ]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏The first time I read I got confused thinking that positive and negative were reversed because for instance the emoji 😡 is a negative feeling. Also I found hard to understand the rating especially with the positive emojis.

What do you think about?

```ts
  const emojiMap = [
    { threshold: 0.15, increase: '😐', decrease: '🙂' },
    { threshold: 0.3, increase: '😠', decrease: '😀' },
    { threshold: 0.5, increase: '😡', decrease: '😃' },
    { threshold: Infinity, increase: '👿 ', decrease: '🤩' },
  ]
```

Reply:
Positive and negative was just here to declare if  "is the bundle size difference positive or negative". But it's maybe clearer to just say increase or decrease. I agree with the scale you made but for the last increase don't you think it would be better to just keep ⛔ ? because in that case we don't want that pr with that much increase

Reply:
⛔  also works :)